

<table>
<colgroup>
<col style="text-align:left;"/>
<col style="text-align:left;"/>
<col style="text-align:left;"/>
<col style="text-align:left;"/>
<col style="text-align:left;"/>
</colgroup>

<thead>
<tr>
	<th style="text-align:left;">Module</th>
	<th style="text-align:left;">Versions</th>
	<th style="text-align:left;">Description</th>
	<th style="text-align:left;">Licenses Available</th>
	<th style="text-align:left;">Notes</th>
</tr>
</thead>

<tbody>
<tr>
	<td style="text-align:left;">bedtools</td>
	<td style="text-align:left;">2.11.2</td>
	<td style="text-align:left;">The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://code.google.com/p/bedtools/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">biostat</td>
	<td style="text-align:left;">1.0</td>
	<td style="text-align:left;">Biostatistics local repository</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://www.sph.umich.edu/biostat/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">bowtie</td>
	<td style="text-align:left;">0.12.7</td>
	<td style="text-align:left;">Bowtie is an ultrafast, memory-efficient short read aligner. It aligns short DNA sequences (reads) to the human genome at a rate of over 25 million 35-bp reads per hour. Bowtie indexes the genome with a Burrows-Wheeler index to keep its memory footprint small</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://bowtie-bio.sourceforge.net/index.shtml">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">bwa</td>
	<td style="text-align:left;">0.5.9</td>
	<td style="text-align:left;">Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://bio-bwa.sourceforge.net/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">macs</td>
	<td style="text-align:left;">1.3.7.1</td>
	<td style="text-align:left;">Next generation parallel sequencing technologies made chromatin immunoprecipitation followed by sequencing (ChIP-Seq) a popular strategy to study genome-wide protein-DNA interactions, while creating challenges for analysis algorithms.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://liulab.dfci.harvard.edu/MACS/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">mmap</td>
	<td style="text-align:left;">20110912</td>
	<td style="text-align:left;">mixed models analysis for pedigrees and populations</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://edn.som.umaryland.edu/mmap/index.php">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">omegamap</td>
	<td style="text-align:left;">0.5</td>
	<td style="text-align:left;">omegaMap is a program for detecting natural selection and recombination in DNA or RNA sequences. It is based on a model of population genetics and molecular evolution. The signature of natural selection is detected using the dN/dS ratio (which measures the relative excess of non-synonymous to synonymous polymorphism) and the signature of recombination is detected from the patterns of linkage disequilibrium.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://www.danielwilson.me.uk/omegaMap.html">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">R</td>
	<td style="text-align:left;">2.10.1 2.14.1</td>
	<td style="text-align:left;">R is a language and environment for statistical computing and graphics.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://www.r-project.org/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">samtools</td>
	<td style="text-align:left;">0.1.13</td>
	<td style="text-align:left;">SAM (Sequence Alignment/Map) format is a generic format for storing large nucleotide sequence alignments.</td>
	<td style="text-align:left;">N/A</td>
	<td style="text-align:left;"><a href="http://samtools.sourceforge.net/">Vendor</a></td>
</tr>
<tr>
	<td style="text-align:left;">sas</td>
	<td style="text-align:left;">9.2</td>
	<td style="text-align:left;">SAS Statistical Analysis package</td>
	<td style="text-align:left;">8</td>
	<td style="text-align:left;"><a href="http://www.sas.com/">Vendor</a></td>
</tr>
</tbody>
</table>

